app/components/Search/Search.module.css

```diff
@@ -1,22 +1,53 @@
 @import url('~app/styles/variables.css');
 
+.wrapper {
+  position: fixed;
+  inset: var(--development-banner-height) 0 0;
```

Comment:
What's the advantages of this approach over what we did previously?

Reply:
The search bar is now aligned with the "close" button, and on development mode it gets pushed further down because of the development banner that says "This is a development version" or something. This is just so that it's always aligned, even on development

Reply:
Ahahh, aight

Reply:
Just to clarify further; the development banner was not visible in the search menu before (see image above). I though it was nice to include :)